lib/ClimaLandSimulations/src/Fluxnet/fluxnet_utilities/inputs_dataframe.jl

```diff
@@ -1,11 +1,25 @@
 export make_inputs_df
 
 """
-    make_inputs_df(LOCAL_DATETIME, drivers)
+    make_inputs_df(site_ID)
 
 Returns DataFrames of inputs, one unitless, one in SI units, one in commonly used units. 
 """
-function make_inputs_df(LOCAL_DATETIME, drivers)
+function make_inputs_df(
+    site_ID;
+    FT = Float64,
+    site_setup_out = site_setup(site_ID),
```

Comment:
could this be "site_setup = make_site_setup"
"domain = make_domain"
"config = make_config"
etc? 

At another point, I had asked about the difference between config and setup. both somewhat vague terms

Reply:
It's also confusing because we are using `setup` as a verb and a noun
"site_setup" (noun)
"setup_drivers" (verb)

Reply:
I'll do the make_setup and make_config, etc., in this PR. 
I agree the code should be refactored a bit to avoid these vague terms (future PR)